documentation/README.md

```diff
@@ -0,0 +1,23 @@
+# Boundary Developer conventions
+
+## I18n:
+
+We use [ember-intl](https://ember-intl.github.io/ember-intl/) to manage i18n. All the code related to i18n lives within [the core addon](https://github.com/hashicorp/boundary-ui/tree/main/addons/core).
+
+All the features provided by emebr-intl are available to use, on top of that, the Boundary team [provides some custom helpers](https://github.com/hashicorp/boundary-ui/tree/main/addons/core/addon/helpers).
```

Comment:
Are the custom helpers really related to internationalization? For the date helpers specifically, in retrospect we should've just used the [date formatter](https://ember-intl.github.io/ember-intl/docs/helpers/format-date) from the addon

Reply:
@ZedLi 
You are right. In retrospect, I arrive to the same conclusion, we could rely within `date formatter`. I will create a low priority ticket, to audit the usage of dates within our UI's with next goals:

- Make sure all the times and dates in our UI are using `<time>` HTML element, [documentation here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) and show timezone.
- Try to rely 100% on the `ember-intl` [date formatter](https://ember-intl.github.io/ember-intl/docs/helpers/format-date).
- Make sure the default or any other format we define within `format.js` shows the timezone.
- Clean up, if need, custom helpers.
- Clean up, if need, formats not used within `format.js` [code here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/formats.js). i.e: I don't think at the moment we are using any `number` ([code here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/formats.js#L38-L52)), or others.
- Update documentation (basicaslly what this PR brings) with the work done if need. I.e: delete from documentation the mention to custom helpers if we end up deprecating them.

Thoughs?